src/common/AndroidUtils.ts

```diff
@@ -709,7 +717,7 @@ export class AndroidSDKUtils {
         const findProcessCommand =
             process.platform === AndroidSDKUtils.WINDOWS_OS
                 ? `tasklist /V /FI "IMAGENAME eq qemu-system-x86_64.exe" | findstr "${emulatorName}"`
-                : `ps -ax | grep 'qemu-system-x86_64 -avd ${emulatorName}' | grep -v 'grep'`;
+                : `ps -ax | grep qemu-system-x86_64 | grep ${emulatorName} | grep -v grep`;
```

Comment:
looks like depending on the version of `qemu-system-x86_64` sometimes the process info shows `qemu-system-x86_64 -avd <name>` and other times it shows `qemu-system-x86_64 @<name>` so modified the `grep` chaining here to handle both cases.